config/default/field.field.taxonomy_term.h_notification_type.field_h_icon.yml

```diff
@@ -0,0 +1,28 @@
+uuid: 183ddbfe-be41-45b2-96ed-b2e512bf4276
```

Comment:
Could you modify two of the three sample notification types to include a custom image. This will help pick up any issues in the notification icon functionality in our visual regression tests.